tools/RAiDER/delay.py

```diff
@@ -78,6 +78,12 @@ def tropo_delay(
         else:
             height_levels = wm_levels
 
+    if isinstance(zref, str):
+        try:
+            zref = float(zref)
+        except:
+            log.warning('Cannot convert zref={%s} to a number', zref)
+
```

Comment:
Don't think you need try/except. Python builtin error will be explicit enough for users.